faker/providers/address/fr_CH/__init__.py

```diff
@@ -0,0 +1,89 @@
+# coding=utf-8
+from __future__ import unicode_literals
+from .. import Provider as AddressProvider
+
+
+class Provider(AddressProvider):
+    city_suffixes = ('-des-Bois', '-les-Bains', '-la-Ville', '-Dessus', 
+        '-Dessous', ' am Rhein', ' am See', ' am Albis', ' an der Aare'
+    )
+    city_prefixes = ('Saint ', 'Sainte ', 'San ', 'Ober', 'Unter')
+    city_formats = (
+        '{{last_name}}',
+        '{{last_name}}',
+        '{{last_name}}',
+        '{{last_name}}',
+        '{{last_name}}{{city_suffix}}',
+        '{{last_name}}{{city_suffix}}',
+        '{{last_name}}{{city_suffix}}',
+        '{{last_name}}-près-{{last_name}}',
+        '{{last_name}}-sur-{{last_name}}',
+        '{{city_prefix}}{{last_name}}',
+        '{{last_name}} ({{canton_code}})',
+    )
+
+    street_address_formats = (
+        '{{street_name}}',
+        '{{street_name}} {{building_number}}',
+        '{{street_name}} {{building_number}}',
+        '{{street_name}} {{building_number}}',
+        '{{street_name}} {{building_number}}',
+        '{{street_name}} {{building_number}}',
+    )
+
+    postcode_formats = ('1###', '2###', '3###', '4###', '5###', '6###', '7###',
+        '8###', '9###'
+    )
+    
+    cantons = (
+        ('AG', 'Argovie'), ('AI', 'Appenzell Rhodes-Intérieures'), 
+        ('AR', 'Appenzell Rhodes-Extérieures'), ('BE', 'Berne'), 
+        ('BL', 'Bâle-Campagne'), ('BS', 'Bâle-Ville'), ('FR', 'Fribourg'),
+        ('GE', 'Genève'), ('GL', 'Glaris'), ('GR', 'Grisons'), ('JU', 'Jura'),
+        ('LU', 'Lucerne'), ('NE', 'Neuchâtel'), ('NW', 'Nidwald'), 
+        ('OW', 'Obwald'), ('SG', 'Saint-Gall'), ('SH', 'Schaffhouse'), 
+        ('SO', 'Soleure'), ('SZ', 'Schwytz'), ('TG', 'Thurgovie'), 
+        ('TI', 'Tessin'), ('UR', 'Uri'), ('VD', 'Vaud'), ('VS', 'Valais'),
+        ('ZG', 'Zoug'), ('ZH', 'Zurich')
+    )
+
+    @classmethod
+    def street_prefix(cls):
+        """
+        :example 'rue'
+        """
+        return cls.random_element(cls.street_prefixes)
```

Comment:
Thanks for your review.
I fixed it in commit fc187ebf1df98f59660fafa77b3d06eff61d06bb. All the tests have passed.


> Le 28 avr. 2016 à 16:16, Flavio Curella <notifications@github.com> a écrit :
> 
> In faker/providers/address/fr_CH/__init__.py <https://github.com/joke2k/faker/pull/348#discussion_r61434232>:
> 
> > +        ('AR', 'Appenzell Rhodes-Extérieures'), ('BE', 'Berne'), 
> > +        ('BL', 'Bâle-Campagne'), ('BS', 'Bâle-Ville'), ('FR', 'Fribourg'),
> > +        ('GE', 'Genève'), ('GL', 'Glaris'), ('GR', 'Grisons'), ('JU', 'Jura'),
> > +        ('LU', 'Lucerne'), ('NE', 'Neuchâtel'), ('NW', 'Nidwald'), 
> > +        ('OW', 'Obwald'), ('SG', 'Saint-Gall'), ('SH', 'Schaffhouse'), 
> > +        ('SO', 'Soleure'), ('SZ', 'Schwytz'), ('TG', 'Thurgovie'), 
> > +        ('TI', 'Tessin'), ('UR', 'Uri'), ('VD', 'Vaud'), ('VS', 'Valais'),
> > +        ('ZG', 'Zoug'), ('ZH', 'Zurich')
> > +    )
> > +
> > +    @classmethod
> > +    def street_prefix(cls):
> > +        """
> > +        :example 'rue'
> > +        """
> > +        return cls.random_element(cls.street_prefixes)
> I don't see the street_prefixes attribute defined anywhere in this class
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/joke2k/faker/pull/348/files/2e81e142499da46a3e3dc8eaaf51a92df475cdb9#r61434232>